test/unit/alignment/multiple/align_multiple_test.cpp

```diff
@@ -84,3 +92,133 @@ TEST(align_multiple_test, the_first_aminoacid_test)
+    };
+
+    constexpr seqan3::configuration cfg = seqan3::align_cfg::gap{seqan3::gap_scheme{seqan3::gap_score{-4},
+                                                                                    seqan3::gap_open_score{-12}}};
+
+    auto result = seqan3::align_multiple(input, cfg);
+
+    EXPECT_RANGE_EQ(output, result);
+}
+
+TEST(align_multiple_test, gap_score_conversion_test)
+{
+    // compute alignment with seqan3::align_multiple interface ---------------------------------------------------------
+    constexpr seqan3::configuration cfg = seqan3::align_cfg::gap{seqan3::gap_scheme{seqan3::gap_score{-1},
+                                                                                    seqan3::gap_open_score{-1}}};
+    std::vector<seqan3::rna4_vector> input{"UUUACCCGAG"_rna4, "UUCCCGCG"_rna4, "UUUACGU"_rna4};
+    seqan3::gap g{};
+    // with go = -1, g = -1
+    auto seqan3_interface_result = seqan3::align_multiple(input, cfg);
+
+    // compute alignment with seqan2------------------------------------------------------------------------------------
+    using alphabet_type = seqan::Rna;
+
+    using sequence_type = seqan::String<alphabet_type>;
+    using graph_type = seqan::Graph<seqan::Alignment<seqan::StringSet<sequence_type, seqan::Dependent<>>,
+                                                     void,
+                                                     seqan::WithoutEdgeId>>;
+
+    seqan::MsaOptions<seqan::Rna, seqan::Score<int>> msaOpt{};
+    msaOpt.build = 0; // NJ for building the guide tree.
+    seqan::appendValue(msaOpt.method, 0); // global alignment
+    seqan::appendValue(msaOpt.method, 1); // local alignment
+    msaOpt.pairwiseAlignmentMethod = 1; // unbanded
+    seqan::Score<int> scMat;
+    msaOpt.sc = scMat;
+    msaOpt.sc.data_gap_open = -2; // seqan2 does not add a gap score for the first gap character but just the go score.
+                                  // seqan3 does add the gap score for the first gap character, so we add -1 to the
+                                  // seqan2 go score.
+    msaOpt.sc.data_gap_extend = -1;
+    msaOpt.sc.data_match = 5;       // tcoffee app default
+    msaOpt.sc.data_mismatch = -4;   // tcoffee app default
+    // fill seqan2 data storage
+    seqan::StringSet<sequence_type, seqan::Owner<>> sequenceSet;
+    seqan::appendValue(sequenceSet, "UUUACCCGAG");
+    seqan::appendValue(sequenceSet, "UUCCCGCG");
+    seqan::appendValue(sequenceSet, "UUUACGU");
+    seqan::StringSet<seqan::String<char>> sequenceNames;
+    seqan::appendValue(sequenceNames, "1");
+    seqan::appendValue(sequenceNames, "2");
+    seqan::appendValue(sequenceNames, "3");
+
+    // Alignment of the sequences
+    graph_type gAlign;
+
+    // MSA
+    seqan::globalMsaAlignment(gAlign, sequenceSet, sequenceNames, msaOpt);
+
+    std::string mat;
+    seqan::convertAlignment(gAlign, mat);
+
+    EXPECT_RANGE_EQ(mat, seqan3_interface_result | seqan3::views::to_char | seqan3::views::join);
```

Comment:
I think it is enough to test the following?
```suggestion
    // with go = -1, g = -1
    constexpr seqan3::configuration cfg = seqan3::align_cfg::gap{seqan3::gap_scheme{seqan3::gap_score{-1},
                                                                                    seqan3::gap_open_score{-1}}};

    // seqan2 does not add a gap score for the first gap character but just the gap open score
    // seqan3 does add the gap extension score additionally to the open score for the first gap character
    auto msaOpt = seqan2_msa_configuration(config);
    EXPECT_EQ(msaOpt.sc.data_gap_extend, -1);
    EXPECT_EQ(msaOpt.sc.data_gap_open, -2);
```